### PR TITLE
Reworking the tech blog so it looks pretty 💝

### PR DESCRIPTION
**Pull request by saibotsivad**
https://eds.kanbanize.com/ctrl_board/2/cards/4898/details

As a parting 🎁 I've gone through the tech blog and updated it to use a standard [Jekyll Blog](https://jekyllrb.com/) (that's what Github Pages uses by default) and to look pretty.

You should be able to run Jekyll locally to see what the output will be, by going to the folder and running:

```
builder exec jekyll serve
# => Now browse to http://localhost:4000
```

image

### CODE REVIEW

**Comment by smithaps89** [comment]
Nice!

Now are you going to leave us with a parting blog post as well 😉

File: _includes/header.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<!--
+This site designed with ♥ by Tobias Davis: http://saibotsivad.com
```
Comment (smithaps89) [inline _includes/header.html]: Little plug here for your site? haha 🐳

Comment (saibotsivad) [inline _includes/header.html]: Haha I didn't notice! I actually copied over the framework from another site of mine.

**Comment by saibotsivad** [comment]
I was thinking to write up a post about our AngularJS build, because I think it's interesting. But I'll make a different pull request for that one 😄

**Pull request merged**